src/App.js

```diff
@@ -1,9 +1,11 @@
 import Calculator from './components/Calculator';
+import Fetch from './components/Quote';
 
 function App() {
   return (
     <div className="App">
       <Calculator />
+      <Fetch />
```

Comment:
- [ ] [OPTIONAL] The quotes would have been placed on top so that users should not scroll to the bottom to see them.